src/compiler/operations.md

```diff
@@ -4,7 +4,7 @@
 
 Here is a list of recurring tasks. Ideally run through this list every week. If there are blockers or doubts, after having acquired the right context, don't hesitate to ping people around. Contributors are the best resource of the project (and we want to be mindful of their time) and are always helpful.
 
-You can trigger a discussion about a specific topic, issue or pull request by opening a new topic on Zulip in [#t-compiler][t-compiler] or, if it needs consensus and more focus from the team, it can can be labeled `I-compiler-nominated` and will be discussed in a meeting (see section [Meetings][meetings]).
+You can trigger a discussion about a specific topic, issue or pull request by opening a new topic on Zulip in [#t-compiler][t-compiler]. If a matter needs more urgent discussion by the team, a new issue can be created and labeled with `I-compiler-nominated` to nominate it for discussion in the next compiler triage meeting (see section [Meetings][meetings]).
```

Comment:
I would remove the "urgency". Yes, it can be a factor but the idea behind the label is to get consensus and focus from us, or something needs more people around the table :)

Also I don't have clear why "opening a new issue"? Usually we apply this label to existing issues/PRs. Basically anything that needs design and it's not a yes/no question.

Reply:
> I would remove the "urgency". Yes, it can be a factor but the idea behind the label is to get consensus and focus from us, or something needs more people around the table :)

Sure.

> Also I don't have clear why "opening a new issue"? Usually we apply this label to existing issues/PRs. Basically anything that needs design and it's not a yes/no question.

Very simple: I had a brainfart and thought it was issue-only for some reason.